packages/table/src/headers/columnHeader.tsx

```diff
@@ -60,6 +65,12 @@ export interface IColumnHeaderProps extends ISelectableProps, IColumnIndices, IC
     viewportRect: Rect;
 
     /**
+     * Enables/disables the reordering interaction.
+     * @default true
+     */
+    isReorderable?: boolean;
```

Comment:
@themadcreator do you mean up to a higher-level component? I'd still need some way to let all the column headers know whether or not to allow reordering though, right? That would seem to require a prop here regardless.